### PR TITLE
Added Kansas State Taxes

### PR DESCRIPTION
**Pull request by cstait**

### CODE REVIEW

File: CertiPay.Taxes.State/Kansas/TaxTable.cs [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+            taxableWages -= GetPersonalAllowance(personalAllowances);
+
+            var selected_row = GetTaxWithholding(filingStatus, taxableWages);
+
+            // Calculate the withholding from the percentages
+
+            var taxWithheld = selected_row.TaxBase + ((taxableWages - selected_row.StartingAmount) * selected_row.TaxRate);
+
+            return Math.Round(frequency.CalculateDeannualized(taxWithheld), MidpointRounding.AwayFromZero);
```
Comment (mattgwagner) [inline CertiPay.Taxes.State/Kansas/TaxTable.cs]: Just for reference:

`CalculateDeannualized` (https://github.com/CertiPay/CertiPay.Payroll.Common/blob/master/CertiPay.Payroll.Common/PayrollFrequency.cs#L151) rounds to even via (https://github.com/CertiPay/CertiPay.Payroll.Common/blob/master/CertiPay.Payroll.Common/ExtensionMethods.cs#L14), so I'm not sure what effect rounding the rounded value ends up with?

File: CertiPay.Taxes.State/Kansas/TaxTable.cs [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+            public Decimal MaximumWage { get; set; }
+
+            public Decimal TaxRate { get; set; }
+        }        
+    }
+    public enum FilingStatus : Byte
+    {
+        [Display(Name = "Single")]
+        Single = 0,
+        [Display(Name = "Married or Head of Household")]
```
Comment (mattgwagner) [inline CertiPay.Taxes.State/Kansas/TaxTable.cs]: The enum should probably line up more closely with the intent -- if it's Married or HoH, either make the name like that or split to two enums?

File: CertiPay.Taxes.State/Kansas/TaxTable.cs (hunk 1 above)
Comment (cstait) [inline CertiPay.Taxes.State/Kansas/TaxTable.cs]: Need to round to the nearest whole dollar, at least in this case.

File: CertiPay.Taxes.State/Kansas/TaxTable.cs (hunk 2 above)
Comment (cstait) [inline CertiPay.Taxes.State/Kansas/TaxTable.cs]: I'll change the enum name, splitting it into another name just means more tables and longer retrieval times (however small they might be).

File: CertiPay.Taxes.State/Kansas/TaxTable.cs (hunk 1 above)
Comment (mattgwagner) [inline CertiPay.Taxes.State/Kansas/TaxTable.cs]: We use this a few other places for states with similar expectations:

https://github.com/CertiPay/CertiPay.Taxes.State/blob/f5bbe1d3367946d228379a777e906b00e67dbc07/CertiPay.Taxes.State/NorthCarolina/TaxTable.cs#L30

**Pull request merged**